ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/workspace-menu/workspace-menu.tsx

```diff
@@ -47,73 +52,101 @@ const Icon = styled.span`
 `
 
 const Root = styled<{ saved: boolean }, 'div'>('div')`
+  padding: ${({ theme }) => theme.minimumSpacing};
+  line-height: calc(
+    2 * ${({ theme }) => theme.minimumLineSize} - 2 *
+      ${({ theme }) => theme.minimumSpacing}
```

Comment:
This might be a bit more clear?
```suggestion
     line-height: calc( 2 * ( ${({ theme }) => ( theme.minimumLineSize - theme.minimumSpacing)})
```